microservices/payment-stripe/src/entities/transaction.ts

```diff
@@ -119,28 +149,34 @@ class Transaction {
   paymentMethodId: string | null;
 
   @JSONSchema({
-    description: 'Microservice entity',
+    description: `Microservice entity. Single line item (ticket, product) or for custom low-level workflow
+    (payment intent) - payment group`,
   })
   @Column({ type: 'varchar', length: 36 })
   @IsUndefinable()
   @Length(1, 36)
   entityId: string;
 
-  @JSONSchema({ description: 'Unit amount (e.g. 100$ = 10000 in unit' })
+  @JSONSchema({
+    description: `Processing amount, includes all fees, taxes. Presented in units,
```

Comment:
why you using ` instead '?

Reply:
There was a lengthy string, but I reduced the description content without returning ''